checkstyle/suppressions.xml

```diff
@@ -154,7 +154,7 @@
               files="(KafkaConfigBackingStore|Values|ConnectMetricsRegistry).java"/>
 
     <suppress checks="NPathComplexity"
-              files="(DistributedHerder|RestClient|RestServer|JsonConverter|KafkaConfigBackingStore|FileStreamSourceTask|WorkerSourceTask|TopicAdmin).java"/>
+              files="(DistributedHerder|RestClient|RestServer|JsonConverter|KafkaConfigBackingStore|FileStreamSourceTask|WorkerSourceTask|TopicAdmin|ConnectorOffsetBackingStore).java"/>
```

Comment:
`ConnectorOffsetBackingStore` has been added here mainly due to the new logic to write to secondary stores first for tombstones. I can remove it if this is not agreed upon.

Reply:
This is not needed anymore.